fieldservice_equipment_warranty/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,12 @@
+Customizations related to ``fieldservice_equipment_stock`` module and ``product_warranty`` module.
+
+Models:
+
+* On ``fsm.equipment`` model:
+   * Add two new fields ``warranty_start_date`` and ``warranty_end_date``.
+   * Override ``create`` method to set the ``warranty_end_date`` field based on warranty duration.
```

Comment:
I think this line can be removed
```suggestion
```

Reply:
Indeed. `README.md` should be readable by functionals, mostly.
Would be great if you can rework it a little bit to focus on the actual purpose of the module, rather than how technically is doing it